assets/script.js

```diff
@@ -2,7 +2,40 @@
 
 $(document).ready(function () { // using ready() method to warp all the codes to make sure the HTML is fully loaded before running any other codes 
   var apiKey = "e9fca33cb368be842d7fa15031d8b7e5"; // puze zhong's openweather API key for weather info
-
+  var APIKey = "TnY7dgkAkdBckSWhls81z22VTZLosfGM"; // Sarah's Amadeus API key for travel info
+  var SECRET_KEY = '294cZnBTO3GbEjQS'
+  function loadFlights(city) {
+    var travelApiUrl = "https://test.api.amadeus.com/v2/shopping/flight-offers?currencyCode=USD&originLocationCode=SYD&destinationLocationCode=BKK&departureDate=2023-05-30&adults=1&nonStop=false&max=10"
+    // SYD&destinationLocationCode=BKK&departureDate=2023-05-02&adults=1&nonStop=false&max=250
```

Comment:
This looks good for now, but we're gonna want to pass in the city code dynamically instead of hardcoding but I'm sure that's the next step.